packages/project-input-validator/package.json

```
@@ -0,0 +1,44 @@
+{
+    "name": "@sap-ux/project-input-validator",
+    "version": "0.1.0",
```

```suggestion
    "version": "0.0.1",
```
Would probably go for this and let release bump it to `0.1.0` instead of `0.2.0`

---

switched back to 0.0.1